### PR TITLE
Porting l1t developments since December to cms-sw

### PR DESCRIPTION
**Pull request by cecilecaillol**
#### PR description:

This PR contains changes done in the l1t branch since December (the minor changes that did not need an immediate integration to cms-sw). 

#### PR validation:

All the PRs were validated in the l1t-offline repository, and have been used for weeks.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

l1t-offline:l1t-integration-CMSSW_11_2_0

@rekovic

### CODE REVIEW

**Comment by jfernan2** [comment]
@cecilecaillol  can you please addyourself along with your github username to the corresponding e-group on
https://twiki.cern.ch/twiki/bin/viewauth/CMS/DQMContacts#L1T
?
Thanks

File: EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc [hunk 1]
```diff
@@ -151,7 +151,7 @@ void GEMRawToDigiModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::Eve
       // best to skip these events since FED is most likely corrupt
       edm::LogWarning("GEMRawToDigiModule")
           << "FED trailer: fail check? " << failTrailerCheck << " fail match? " << failTrailerMatch;
-      continue;
+      //  continue;
```
Comment (dildick) [inline EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc:154]: @jshlee Attention: is this something that should be in the 11_3_X?

Comment (jshlee) [reply EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc:154]: it's best to keep the continue.
or we can make it python configurable to skip bad FEDs

Comment (slava77) [reply EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc:154]: @cecilecaillol @jshlee 
please clarify what will be done here.
It looks like this edit is the only reason for the reco signature.
Thank you.

Comment (cecilecaillol) [reply EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc]: I have kept the continue in the new PR. I hope this is ok for you now.

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
@cecilecaillol  is this going to be needed in the next MWGRs?

**Comment by jfernan2** [comment]
> 
> 
> @cecilecaillol is this going to be needed in the next MWGRs?

@cecilecaillol  if so, a backport to 11_2_X is needed in order to test it inv DQM Online at P5. Thanks

**Comment by boudoul** [comment]
Hello I have the impression that there is a mixture of code that would be good to have in MWGR but not everything. This is what I was asking at the ORP :  what is intented to be used in the next MWGR , what is not . ... We may want to be careful and not backport in a production release a big thing while only a fraction is needed. But of course @silviodonato and @qliphy can decide. Anyhow the first thing to get  is indeed a better description of the content.
@amassiro FYI

**Comment by cecilecaillol** [comment]
> > @cecilecaillol is this going to be needed in the next MWGRs?
> 
> @cecilecaillol if so, a backport to 11_2_X is needed in order to test it inv DQM Online at P5. Thanks

We do not need anything back-ported to 11_2_X. Everything necessary for MWGR was PRed individually before.

**Comment by jfernan2** [comment]
+1

**Comment by rekovic** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**